grails-app/controllers/org/broadinstitute/orsp/api/InfoLinkController.groovy

```diff
@@ -0,0 +1,33 @@
+package org.broadinstitute.orsp.api
+
+import com.google.gson.Gson
+import grails.converters.JSON
+import grails.rest.Resource
+import groovy.util.logging.Slf4j
+import org.broadinstitute.orsp.AuthenticatedController
+
+@Slf4j
+@Resource(readOnly = false, formats = ['JSON', 'APPLICATION-MULTIPART'])
+class InfoLinkController extends AuthenticatedController {
+
+    def showInfoLink() {
+        render(view: "/infoLink/index")
+    }
+
+    def getProjectSampleCollections() {
+        String consentKey = params.consentKey
+        String projectKey = params.projectKey
+        try {
+            Gson gson = new Gson()
+            def result = queryService.findCollectionLinksByConsentKeyAndProjectKey(consentKey, projectKey)
```

Comment:
This should be a typed object, not a `def`. I think you want `Map<ConsentCollectionLinkDTO, List<StorageDocument>>` here.